services/server/src/server/services/utils/contract-creation-util.ts

```diff
@@ -243,6 +243,7 @@ async function getCreatorTxUsingFetcher(
     }
   } catch (e: any) {
     logger.warn("Error while getting creation transaction", {
+      url: fetcher?.url,
```

Comment:
This will log API keys (currently Etherscan and VeChain). This is especially bad since CI logs are public. Let's either prune the keys before logging, or not log the URL.

Reply:
Ah after looking at the code I saw that we actually skip these tests for that reason. I still think it's not great that we log the keys in GCP, but not blocking for this PR.